packages/p2p/src/components/modal-manager/modals/create-ad-modal/create-ad-modal-card/create-ad-modal-card.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { Button, Text, Icon } from '@deriv/components';
+import { Localize } from 'Components/i18next';
+import { useStores } from 'Stores';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+import BlockTradeAd from 'Assets/svgComponents/block-trade-ad.svg';
+import './create-ad-modal-card.scss';
+
+const CreateAdModalCard = ({ title, text, type }) => {
+    const { hideModal } = useModalManagerContext();
+    const { my_ads_store } = useStores();
+
+    const onClickChoose = () => {
+        my_ads_store.setAdType(type);
+        my_ads_store.onClickCreate();
+        hideModal();
+    };
+
+    return (
+        <div className='ad-component'>
+            {type === 'normal' ? <Icon icon='IcCashierNormalAd' width={48} height={48} /> : <BlockTradeAd />}
```

Comment:
I think its better to do it like this rather than to import the icon. When you have managed to move the icon to the correct space could you right it like this:

```suggestion
            <Icon icon={type === 'normal' ? 'IcCashierNormalAd' : 'IcBlockTradeAd'} width={48} height={48} />
```